htdocs/core/modules/modCommande.class.php

```diff
@@ -262,6 +263,161 @@ public function __construct($db)
 		$this->export_sql_end[$r] .= ' WHERE c.fk_soc = s.rowid AND c.rowid = cd.fk_commande';
 		$this->export_sql_end[$r] .= ' AND c.entity IN ('.getEntity('commande').')';
 		if (empty($user->rights->societe->client->voir)) $this->export_sql_end[$r] .= ' AND sc.fk_user = '.(empty($user) ? 0 : $user->id);
+		// Imports
```

Comment:
As this is a new feature, it has been merged for v13 and will be added in v13 only.

Reply:
> As this is a new feature, it has been merged for v13 and will be added in v13 only.

Hi,
do you mean everyhing or just propal/order/invoice ?
OK, so no new features on 12.0.x. Could it be pushed to a hypothetical 12.1 ?

Other fields were exported but just missing on import, so it's not really a new feature.
I modified the PR accordingly.

Reply:
If 12.1 is 12 + new features, we can But we have just called it 13.0  :-)
If it is adding existing feature not exported, we can say it is a bug.
If it changes the algorithm how we load data, it is better in a new version.

Reply:
> If 12.1 is 12 + new features, we can But we have just called it 13.0 :-)

There is a 10.5 version, but it seems to be the only one like this. I thought minor improvements could be added also to minor revisions of major maintained versions. So OK in x.y.Z you actually don't use Y but only Z for bug fixes.
Then I looked to the history and found [RoadMap](https://wiki.dolibarr.org/index.php?title=Category:RoadMap) and [Release Process](https://wiki.dolibarr.org/index.php?title=FAQ_Release_process)